MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/IEnggDiffractionView.h

```diff
@@ -343,7 +351,23 @@ class IEnggDiffractionView {
  *
  * @return bool
  */
-  virtual bool saveOutputFiles() const = 0;
+  virtual bool saveFocusedOutputFiles() const = 0;
+
+  /**
+  * Produces vanadium curves graph with three spectrum for calib
+  * output. Runsm plotSpectrum function via python.
+  *
+  */
+  virtual void plotVanCurvesCalibOutput() = 0;
+
+  /**
+  * Produces ceria peaks graph with two spectrum for calib
+  * output. Runs the plotSpectrum function via python.
```

Comment:
Yes, this is about the IEnggDifffractionView header, which is supposed to be an abstract interface, and which for example could be used in a totally different, say web based, interface. This is just a nit-picky thing. You can remove them or say that that's they way it is done for example in the Qt view, or similar.